en/docs/learn/multi-factor-authentication-for-wso2-is-management-console.md

```diff
@@ -22,17 +22,25 @@
     
     ![enable-response-signing](../assets/img/using-wso2-identity-server/enable-response-signing.png)
 
-6.  Expand **Local and Outbound Authentication Configuration** and select **Advanced Configuration** to configure MFA. There are two types of MFA that you can configure here.
+6.  Expand **Local & Outbound Authentication Configuration** and select **Advanced Configuration** to configure MFA. There are two types of MFA that you can configure here.
+
+    ![add-auth-step](../assets/img/using-wso2-identity-server/add-auth-step.png)
```

Comment:
There is no image filename with the name "add-auth-step.png". Did you mean "add-authentication-step.png"? 

Also - right after the line "There are two types of MFA that you can configure here", the user will expect to see the two options instead of an image. Is this image really required here? If not, shall we remove it?